xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -768,7 +768,7 @@ JSONRPC_STATUS JSONSchemaTypeDefinition::Check(const CVariant &value, CVariant &
       // allowed there is no need to check every element
       if (value.size() < items.size() || (value.size() != items.size() && additionalItems.size() == 0))
       {
-        CLog::Log(LOGDEBUG, "JSONRPC: One of the array elements does not match in type {0}", name.c_str());
```

Comment:
You're right, I didn't look close enough. Since the PR description mentioned `StringUtils::Format()` I assumed this was a `StringUtils::Format()` call as well. In that case this change is correct.